CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [3.1.0] - April 9, 2024
+
+### Bug Fixes
+- For Issues [#235](https://github.com/optimizely/react-sdk/issues/235) & [#236](https://github.com/optimizely/react-sdk/issues/236) ([#255](https://github.com/optimizely/react-sdk/pull/255))
```

Comment:
We usually have bug fix summaries here with PR links (and each PR will have a link to issue).